src/View/PasswordQuickInput.ts

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (c) 2022 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as vscode from 'vscode';
+
+export async function showPasswordQuickInput(): Promise<string|undefined> {
+  return await vscode.window.showInputBox({
+    title: 'Enter password',
+    password: true,
+    validateInput: validateInputIsAscii,
+  });
+
+  async function validateInputIsAscii(str: string) {
+    await new Promise(resolve => setTimeout(resolve, 1000));
+    // NOTE ASCII characters have codes ranging from u+0000 to u+007f
+    return !/^[\u0000-\u007f]*$/.test(str) ? 'Password contains non-ASCII characters' : undefined;
```

Comment:
I was curious what `007f` was. I found that `DELETE` was `007f`. 
Question is 'is it OK with this validation rule (DELETE is not a char)'? I guess that we will ask Ubuntu `sudo` if this password is correct and `sudo` will check again. 
So I think this check is weak but there's no problem.

Reply:
You can see the detail information about ASCII characters.
https://en.wikipedia.org/wiki/Basic_Latin_(Unicode_block)#:~:text=The%20block%20contains%20all%20the,alphabet%20and%20a%20control%20character.

another examples:
https://thewebdev.info/2022/01/25/how-to-check-if-a-string-has-any-non-ascii-characters-in-it-with-javascript/
https://stackoverflow.com/questions/14313183/javascript-regex-how-do-i-check-if-the-string-is-ascii-only